docker-compose.dev-deps.yml

```diff
@@ -14,6 +14,20 @@ services:
     volumes:
       - './data/mongo:/data/db'
 
+  postgres:
+    image: postgres:17
+    container_name: postgres
+    restart: always
+    environment:
+      POSTGRES_USER: postgres
+      POSTGRES_PASSWORD: postgres
+      POSTGRES_DB: postgres
+    ports:
+      - '5432:5432'
+    volumes:
+      - ./data/postgres:/var/lib/postgresql/data
+      - ./packages/migration/src/migrations/postgres/0001-init-postgres.sql:/docker-entrypoint-initdb.d/0001-init-postgres.sql
```

Comment:
A different mechanism for running migrations with the 'postgres' super user is likely needed - what if we need to add a new database for example? The Docker entrypoint runs only once, when the data folder does not exist yet.